doc/sf/ccmath.qbk

```diff
@@ -51,6 +51,12 @@ All of the following functions require C++17 or greater.
         template <typename T>
         inline constexpr int fpclassify(T x);
 
+        template <typename Real>
+        inline constexpr Real frexp(Real arg, int* exp);
+
+        template <typename Z>
```

Comment:
Trivial quibble, but I when I first read this I didn't grok that this was limited to integer arguments, maybe use `<typename Integer>` and perhaps a comment that this is for integer arguments?

Other than that, I'm happy for this to be merged, and you'll be pleased to know that ldexp should be noticeably easier! :)

Reply:
@jzmaddock : That was my (perhaps poor) convention; mimicking ℤ.

Is it possible to do a concept without breaking C++17 compatibility?

Reply:
Not really, although the enable_if does much the same thing.  It's debatable whether the enable_if condition should be part of the declared interface - it clutters up the docs, but does at least make it clear what the requirements on Z are.

Reply:
Doc suggestions have been incorporated in the latest commit.

As for concepts, you can avoid breaking C++17 compatibility using this macro (taken from [good concepts](https://www.stroustrup.com/good_concepts.pdf)): 

```
#ifdef GOOD_COMPILER 
#define REQUIRES requires 
#elseif 
#define REQUIRES // 
#endif 
```